redbeams/src/main/java/com/sequenceiq/redbeams/configuration/DatabaseServerSslCertificateConfig.java

```diff
@@ -220,24 +263,24 @@ private void logCerts() {
             int minVersion = certMinVersionsByCloudPlatformCache.get(cloudPlatform);
             int maxVersion = certMaxVersionsByCloudPlatformCache.get(cloudPlatform);
             LOGGER.info("Found SSL certificates registered for cloud platform \"{}\": numberOfCerts={}, minVersion={}, maxVersion={}, certs=\"{}\"",
-                    cloudPlatform, numCerts, minVersion, maxVersion, entry.getValue());
+                    prettyPrintCloudPlatform(cloudPlatform), numCerts, minVersion, maxVersion, entry.getValue());
         }
         LOGGER.info("Total number of SSL certificates registered: {}", numCertsTotal);
     }
 
-    public SslCertificateEntry getCertByPlatformAndVersion(String cloudPlatform, int version) {
-        Set<SslCertificateEntry> result = getCertsByPlatformAndVersions(cloudPlatform, version);
+    public SslCertificateEntry getCertByPlatformAndVersion(String cloudPlatform, String region, int version) {
+        Set<SslCertificateEntry> result = getCertsByPlatformAndVersions(cloudPlatform, region, version);
         return result.isEmpty() ? null : result.iterator().next();
     }
 
-    public Set<SslCertificateEntry> getCertsByPlatformAndVersions(String cloudPlatform, int... versions) {
+    public Set<SslCertificateEntry> getCertsByPlatformAndVersions(String cloudPlatform, String region, int... versions) {
```

Comment:
nit: `getCertsByPlatformAndRegionAndVersions`

Reply:
Makes sense. I have also renamed all the other `public` methods in a similar way.